packages/modeling/src/operations/extrusions/extrudeRotate.js

```diff
@@ -114,7 +114,11 @@ const extrudeRotate = (options, geometry) => {
 
   const matrix = mat4.create()
   const createSlice = (progress, index, base) => {
-    const Zrotation = rotationPerSlice * index + startAngle
+    let Zrotation = rotationPerSlice * index + startAngle
+    // fix rounding error when rotating 2 * PI radians
+    if (totalRotation === Math.PI * 2 && index === segments) {
```

Comment:
Probably very little performance impact but wondering if we could improve this.
- Add PI2 constant to maths/constants.js
- Set isFullRotation boolean above, and use here

Reply:
Pi was a mistake, it always should have been Tau. :-)
https://tauday.com/tau-manifesto

Reply:
> Pi was a mistake, it always should have been Tau. :-)
> https://tauday.com/tau-manifesto

😎 A very interesting debate with a lot of merit, but changing to Tau is like asking the USA to use metric measurements. 😃